upgrading/topics/keycloak/changes.adoc

```diff
@@ -2,6 +2,28 @@
 
 === Migrating to 11.0.0
 
+==== Upgrade to Wildfly 20
+
+The {project_name} server was upgraded to use Wildfly 20 as the underlying container. This does not directly involve any
+specific {project_name} server functionality, but there are few changes related to the migration, which worth mentioning.
+
+Dependency updates::
+  The dependencies were updated to the versions used by Wildfly 20 server. For example, Infinispan is now 10.1.8.Final.
+
+Configuration changes::
+  There are few configuration changes in the `standalone(-ha).xml` and `domain.xml` files. You should follow the <<_install_new_version>>
+  section to handle the migration of configuration files automatically.
+
+Cross-Datacenter Replication changes::
+  * You will need to upgrade {jdgserver_name} server to version {jdgserver_version}. The older version may still work, but it is
+  not guaranteed as they are not tested anymore.
+  * It is recommended to use the `protocolVersion` property added tp the `remote-store` element when configuring Infinispan caches.
```

Comment:
@mposolda Sorry, but there's a typo (result of a click to right next keyboard key probably) yet. I think it should be `property added to the` instead of `added tp the` current form. WDYT? (sorry for being a way too much pedantic)

Reply:
Updated. Thanks for the catch!